lightspeed-stack_local.yaml

```diff
@@ -0,0 +1,16 @@
+name: Ansible Lightspeed Intelligent Assistant
```

Comment:
Since light-speed-stack config YAML file does not seem to support variable substitution using environment variables, I created a separate file for local execution.